.ci_support/mpi4pywrapper.py

```
@@ -0,0 +1,123 @@
+        self.rank = comm.rank
+        self.parent = parent  # XXX check C-object against comm.parent?
+
+    def new_communicator(self, ranks):
+        comm = self.comm.Create(self.comm.group.Incl(ranks))
+        if self.comm.rank in ranks:
+            return MPI4PYWrapper(comm, parent=self)
+        else:
+            # This cpu is not in the new communicator:
+            return None
+
+    def max_scalar(self, a, root=-1):
```

Ruff (0.13.3)

Unused method argument: `root`

(ARG002)